src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,400 @@
+---
+section: Guides
+chapter: General
+title: Learning Resources
+description: Where to find out how to learn about the various aspects of NUbots.
+slug: /guides/general/learning-resources
+---
+
+This page aims to provide direction and resources on where to go to learn more about various areas of NUbots. There is a mixture of resources available, including external resources and material prepared by NUbots team members.
+
+NUbots workshops may be run live, or you can work through them yourself in your own time.
+
+The majority of resources on this page are free. Some LinkedIn Learning videos are linked here - UoN students can get access to LinkedIn Learning for free.
+
+## General
+
+<details>
+<summary>Git</summary>
+
+The NUbots team uses Git and GitHub in its development workflow. Almost any project done on the NUbots team involves using Git and GitHub.
+
+**Reading:**
+
+- [Git and PR Workflow](/guides/general/git) [NUbook]
+
+**Tutorials:**
```

Comment:
Do we want to link the git part of the video from the online session we did last year?

Reply:
I'll add this one in, but I think that'll be the last resource to add in for this PR. Further stuff in other PRs, focusing on each section.